src/components/chart/common-chart-options.js

```diff
@@ -171,10 +175,16 @@ class CommonChartOptions {
                         fontSize: this.constants.mobileFontSize,
                     },
                 },
+                tickInterval: yAxisTickInterval,
             },
         };
     };
 
+    updateDesktopTickIntervals = (config, xAxisTickInterval, yAxisTickInterval) => {
```

Comment:
Suggestion: rather than have this in a separate function, just set it in the options for the x axis and y axis in the `this.option` object above - it's more consistent with the other options that way.